roles/nginx/tasks/redhat.yml

```diff
@@ -0,0 +1,40 @@
+- name: Install nginx
+  yum:
+    name:
+      - nginx
```

Comment:
nit: this looks off. with_items?

Reply:
Yum works better if you pass all the packages in at once. Apt doesn't allow this, but yum does.